onecodex/viz/_distance.py

```diff
@@ -289,8 +290,11 @@ def plot_mds(
         from sklearn import manifold
         from sklearn.metrics.pairwise import euclidean_distances
 
-        if len(self._results) < 2:
-            raise OneCodexException("`plot_mds` requires 2 or more valid classification results.")
+        if len(self._results) < 3:
```

Comment:
Very small nitpick: If you would put `3` to constant then you can re-use it in error's message:

```python
MIN_SAMPLES_REQUIRED = 3  # Needs better name
if len(self._results) < MIN_SAMPLES_REQUIRED:
    raise PlottingException(f"... select {MIN_SAMPLES_REQUIRED} or more ...")
```

Feel free to ignore this comment. It really does not need to be changed.